hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ResponseSegment.java

```diff
@@ -3,17 +3,22 @@
 import java.util.Optional;
 
 import ca.bc.gov.hlth.hnsecure.parsing.Util;
+import ca.bc.gov.hlth.hnsecure.properties.ApplicationProperties;
+import ca.bc.gov.hlth.hnsecure.properties.ApplicationProperty;
 
 public abstract class ResponseSegment {
-	private static final String ack = "ACK";
 
-	private static final String fieldSeperator = "|";
+	protected static final String FIELD_SEPARATOR = "|";
 
-	private static final String encodingChar = "^~\\&";
+	private static final String ENCODING_CHARACTERS = "^~\\&";
```

Comment:
Could move the encoding chars constant to Util.java or V2MessageUtil.java for reuse as it's commonly used.

Reply:
Moved to Util. Also removed version in PayloadValidator.